  public void testCuriousCase2() throws Exception {
    verify("u-l-p", CATENATE_ALL);
  }

